warehouse/warehouse.go

```diff
@@ -1393,8 +1380,28 @@ func (wh *HandleT) Setup(whType string) {
 	wh.tenantManager = multitenant.Manager{
 		BackendConfig: backendconfig.DefaultBackendConfig,
 	}
+	whManager, err := manager.New(wh.destType)
```

Comment:
We are not using this `whManager`. Can we remove this since `NewUploadJob` inject the manager itself into `UploadJobT`?